src/ensembl/package.json

```diff
@@ -27,11 +27,9 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "deploy-docs": "./deploy-docs.sh",
     "check-types": "tsc",
-    "jest": "jest",
-    "test": "jest --verbose",
-    "test:watch": "jest --watch",
-    "coverage": "jest --coverage",
-    "snapshot": "jest --updateSnapshot"
+    "test": "jest --verbose --maxWorkers=1",
+    "test:watch": "jest --watch --maxWorkers=1",
```

Comment:
Do we still care about the maxWorkers option? We added it for ts-jest, but now that we switch to babel-jest, is it still relevant?

Reply:
Yes, when I tested without it the test suite runs slower.